### PR TITLE
chore(readme): Add kfp-tekton v2 branch link

### PR DESCRIPTION
**Pull request by Tomcli**
**Which issue is resolved by this Pull Request:** 
Resolves #

**Description of your changes:**

**Environment tested:**

* Python Version (use `python --version`):
* Tekton Version (use `tkn version`):
* Kubernetes Version (use `kubectl version`):
* OS (e.g. from `/etc/os-release`):

**Checklist:**
- [x] The title for your pull request (PR) should follow our title convention. [Learn more about the pull request title convention used in this repository](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md#pull-request-title-convention).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,10 @@ We are currently using [Kubeflow Pipelines 1.8.4](https://github.com/kubeflow/pi
 [Tekton >= 0.47.1](https://github.com/tektoncd/pipeline/releases/tag/v0.47.1)
 for this project.
 
+For [Kubeflow Pipelines 2.0.0](https://github.com/kubeflow/pipelines/releases/tag/2.0.0) and
+[Tekton >= 0.47.1](https://github.com/tektoncd/pipeline/releases/tag/v0.47.1)
```
Comment (yhwang) [inline README.md:22]: the v2-integration branch is using 0.47.3

Comment (Tomcli) [reply README.md:22]: thanks changed it.

**Review by yhwang — APPROVED** [review]
/lgtm

**Pull request merged**